src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurCashAdvancePdpFeedFileServiceImpl.java

```diff
@@ -51,32 +53,73 @@ public boolean createPdpFeedFileForValidatedDetailFileLines(ConcurRequestExtract
         return pdpFileSuccessfullyCreated;
     }
 
-    private PdpFeedFileBaseEntry buildPdpFeedBaseEntry(ConcurRequestExtractFile requestExtractFile) {
+    private PdpFeedFileBaseEntry buildPdpFeedBaseEntry(ConcurRequestExtractFile requestExtractFile, ConcurRequestExtractBatchReportData reportData) {
```

Comment:
this seems like a long function, can this be refactored? to make shorter methods and remove some duplication